e2e/alphabet-card-detail.test.ts

```diff
@@ -47,5 +47,23 @@ describe('Detail', () => {
         await expect(element(by.id('w1.mp3'))).toBeVisible();
     });
 
-    //TODO create a test for error handling if audio is not available
+    it('should display an error message if there is a word audio error', async () => {
+        const expectAudioError = true;
+
+        if (expectAudioError) {
+            await expect(element(by.id('wordAudioError'))).toBeVisible();
```

Comment:
You set `expectAudioErrorToTrue` in this scope, so this entire block is equivalent to one line
```ts
     await expect(element(by.id('wordAudioError'))).toBeVisible();
```

Reply:
Refactored